src/lib/bootloader/sections.rb

```diff
@@ -48,7 +48,9 @@ def write
 
   private
 
-    def read_default
+    # @return [String, nil] return default boot path as string or nil if not set or something goes wrong
+    # @note shows error popup if calling grub2-editenv failed
```

Comment:
ah, this note should in fact be added also to the docs of `on_target` and `locally` https://github.com/yast/yast-yast2/blob/master/library/system/src/lib/yast2/execute.rb#L41